web/app/js/components/EdgesTable.jsx

```diff
@@ -82,21 +83,21 @@ const edgesColumnDefinitions = (PrefixedLink, namespace, type, classes) => {
   ];
 };
 
-const generateEdgesTableTitle = edges => {
+const generateEdgesTableTitle = (edges, t) => {
   let title = 'Edges';
   if (edges.length > 0) {
     let identity = edges[0].direction === 'INBOUND' ? edges[0].serverId : edges[0].clientId;
     if (identity) {
       identity = `${identity.split('.')[0]}.${identity.split('.')[1]}`;
-      title = `${title} (Identity: ${identity})`;
+      title = `${title} (${t('Identity')}: ${identity})`;
     }
   }
   return title;
 };
 
-const EdgesTable = ({ edges, api, namespace, type, classes }) => {
-  const edgesColumns = edgesColumnDefinitions(api.PrefixedLink, namespace, type, classes);
-  const edgesTableTitle = generateEdgesTableTitle(edges);
+const EdgesTable = ({ edges, api, namespace, type, classes, t }) => {
```

Comment:
As this is a functional component right now, I'd recommend using the hooks instead (`useTranslation`). That way, you can use it in the sub-functions as well such as `generateEdgesTableTitle` and `edgesColumnDefinitions` without passing `t` everywhere. It has the added benefit of reducing the component wrapping!